frc2357/robotlib2357
# Add base source structure and build

This PR is the start of a top-level source structure for the library. The `common` directory remains for now, but all code assets will be organized into the new `src/main/java/com/systemmeltdown/robotlib` tree.

This creates two standard source trees, one for the main code, and one for the test code.
It also begins to standardize our package names with a `com.systemmeltdown` prefix because we own the internet domain.

To Test:
1. Hit Ctrl-P and type ">WPILib: Build Robot Code" to build the library.
2. Hit Ctrl-P and type ">WPILib: Test Robot Code" to run the tests.

Verify both steps above run successfully.
